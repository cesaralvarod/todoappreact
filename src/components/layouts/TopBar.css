.container-topbar {
  width: 100%;
  padding: 2rem;
  background: #4d5154;
  color: #fff;
}
