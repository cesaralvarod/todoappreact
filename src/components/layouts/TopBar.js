import React from "react";

// Styles

import "./TopBar.css";

const TopBar = () => {
  return (
    <header className="container-topbar">
      <h1>TodoApp</h1>
    </header>
  );
};

export default TopBar;
