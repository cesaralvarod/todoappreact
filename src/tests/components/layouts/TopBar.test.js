import React from "react";
import { shallow } from "enzyme";
import TopBar from "../../../components/layouts/TopBar";

describe("Testing in component layout TopBar", () => {
  test("Component layout TopBar should show correctly", () => {
    const wrapper = shallow(<TopBar />);

    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find("h1").exists()).toBeTruthy();
  });
});
