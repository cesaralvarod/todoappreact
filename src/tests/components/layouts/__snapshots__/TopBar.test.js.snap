// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing in component layout TopBar Component layout TopBar should show correctly 1`] = `
<header
  className="container-topbar"
>
  <h1>
    TodoApp
  </h1>
</header>
`;
